test/integration/user/test/viewportFeatures.test.js

```diff
@@ -97,6 +98,27 @@ describe('viewportFeatures', () => {
         });
     });
 
+    it('should get interactivity-like features, with vizProperties', done => {
+        layer1.on('loaded', () => {
+            // 1. keep required properties
+            const expected = [
+                { value: 10, category: 'a' },
+                { value: 1000, category: 'b' }
+            ];
+            const featureList = viz1.variables.list.eval();
+            checkFeatures(featureList, expected);
+
+            // 2. and featureVizProperties / variables are available
+            featureList.forEach(feature => {
+                FEATURE_VIZ_PROPERTIES.forEach(propertyName => {
+                    expect(feature.hasOwnProperty(propertyName)).toBeTruthy();
```

Comment:
This checks that the viz properties exist, but it doesn't test they work. I think that a `render` test would be better: wait for `loaded`, read viewportFeatures variable, use a `blendTo` in one of the features, wait, mark the test as done

Reply:
Yeah, that's true. I'll keep this one and add another render test